actions.jsx

```diff
@@ -70,14 +70,19 @@ export function setReferenceValueForIndicator(indicatorId, referenceValue) {
         'boundary_type': 3,
       }),
       xhrFields: {
-        withCredentials: true
+        withCredentials: true,
       },
       /* eslint-enable */
-      success: (data) => {
-        return data;
+      success: (data, textStatus, xhr) => {
+        if (xhr.status === 200) {
+          return data;
+        }
+        else {
+          dispatch(showError('Instellen referentiewaarde mislukt...', 'error'));
+        }
       },
       error: (error) => {
-        dispatch(showError(error.responseJSON.message, 'error'));
+        console.log(error.responseJSON);
```

Comment:
`dispatch(showError(msg))` laat de foutmelding zien aan de gebruiker. De console.log() is alleen nodig tijdens development en wordt bij de production build niet overgenomen is het idee.